config/configmap/inferenceservice.yaml

```diff
@@ -50,6 +84,67 @@ data:
            "s3CABundle": ""
        }
     }
+  # ====================================== INGRESS CONFIGURATION ======================================
+  #  {
+       # ingressGateway specifies the ingress gateway to serve external traffic.
+       # The gateway should be specified in format <gateway namespace>/<gateway name>
+       # NOTE: This configuration only applicable for serverless deployment with Istio configured as network layer.
+  #    "ingressGateway": "knative-serving/knative-ingress-gateway",
+
+       # ingressService specifies the hostname of the ingress service.
+       # NOTE: This configuration only applicable for serverless deployment with Istio configured as network layer.
+  #    "ingressService": "istio-ingressgateway.istio-system.svc.cluster.local",
+
+       # localGateway specifies the gateway which handles the network traffic within the cluster.
+       # NOTE: This configuration only applicable for serverless deployment with Istio configured as network layer.
+  #    "localGateway": "knative-serving/knative-local-gateway",
+
+       # localGatewayService specifies the hostname of the local gateway service.
+       # NOTE: This configuration only applicable for serverless deployment with Istio configured as network layer.
+  #    "localGatewayService": "knative-local-gateway.istio-system.svc.cluster.local",
+
+       # ingressDomain specifies the domain name which is used for creating the url.
+       # If ingressDomain is empty then example.com is used as default domain.
+       # NOTE: This configuration only applicable for raw deployment.
+  #    "ingressDomain": "example.com",
+
+       # ingressClassName specifies the ingress controller to use for ingress traffic.
+       # This is optional and if omitted the default ingress in the cluster is used.
+       # https://kubernetes.io/docs/concepts/services-networking/ingress/#default-ingress-class
+       # NOTE: This configuration only applicable for raw deployment.
+  #    "ingressClassName": "istio",
+
+       # domainTemplate specifies the template for generating domain/url for each inference service by combining variable from:
+       # Name of the inference service  ( {{ .Name}} )
+       # Namespace of the inference service ({{ .Namespace }} )
+       # Annotation of the inference service ({{ .Annotations.key }})
+       # Label of the inference service ({{ .Labels.key }})
+       # IngressDomain ({{ .IngressDomain }})
+       # If domain template is empty the default template {{ .Name }}-{{ .Namespace }}.{{ .IngressDomain }} is used.
+       # NOTE: This configuration only applicable for raw deployment.
+  #    "domainTemplate": "{{ .Name }}-{{ .Namespace }}.{{ .IngressDomain }}",
+
+       # urlScheme specifies the url scheme to use for inference service and inference graph.
+       # If urlScheme is empty then by default http is used.
+       # NOTE: This configuration only applicable for raw deployment.
```

Comment:
I thought this should work for serverless mode as well

Reply:
Yes, you are right. I have updated the docs